[cast] Multicall new (#98)
foundry-rs/starknet-foundry: docs/src/starknet/multicall.md

## Overview

Starknet Foundry cast supports executing multiple deployments or calls with the `sncast multicall run` command.

You need to provide a **path** to a `.toml` file with declarations of desired operations that you want to execute.

> It also help you composing a config `.toml` file with the `sncast multicall new` command.

Reviewer: ```suggestion
You can also compose such config `.toml` file with the `sncast multicall new` command.
```

Author: [address dox fdbck](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/722855d7bb207da35ea14576b497377f59274d74)
